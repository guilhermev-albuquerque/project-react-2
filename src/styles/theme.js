export const theme = {
  colors: {
    mainBg: 'darkred',
    secondaryBg: 'blue',
  },
  fonts: {},
};
