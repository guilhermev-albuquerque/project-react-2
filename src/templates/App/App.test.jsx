import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { theme } from '../../styles/theme';
import Home from './index';

test('renders learn react link', () => {
  renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', {
    name: 'Dale',
  }).parentElement;
  expect(headingContainer).toHaveStyle({ background: theme.colors.mainBg });
  expect(headingContainer).toHaveStyleRule('background', 'darkred');
  expect(headingContainer).toMatchSnapshot();
});
