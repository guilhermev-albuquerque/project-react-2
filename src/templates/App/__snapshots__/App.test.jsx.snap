// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  background: darkred;
}

<div
  class="c0"
>
  <h1>
    Dale
  </h1>
</div>
`;
